packages/@dcl/ecs/src/systems/crdt/index.ts

```diff
@@ -161,14 +161,6 @@ export function crdtSceneSystem(engine: PreEngine, onProcessEntityComponentChang
 
         /* istanbul ignore else */
         if (component) {
-          if (
```

Comment:
You cant delete this line since it will break the syncronization of that entity (and its parent).

Reply:
Okay, I understood your point that we need it zero parent to avoid conflicts while syncing with other players.
Moreover, it was not the root cause of the bug, so this is not actually a 'fix' anymore, you've got a better one.
Nevertheless, I'm up to propose a more consistent way of doing this, I'll update this PR as soon as your fix got merged.